### PR TITLE
Added a --strict-redirect-location option

### PR DESCRIPTION
**Pull request by jacobm-splunk**
Added a --strict-redirect-location option; if enabled wiretap will rewrite all `Location` header hosts to wiretap's api gateway host

### CODE REVIEW

**Review by daveshanley — APPROVED** [review]
Nice upgrade! thank you!

**Comment by jacobm-splunk** [comment]
@daveshanley I had a small bug I realized I had when thinking about it. Just pushed a fix.

File: cmd/root_command.go [hunk 1]
```diff
@@ -359,11 +368,6 @@ var (
 				printLoadedWebsockets(config.WebsocketConfigs)
 			}
 
-			if len(config.ValidationAllowList) > 0 {
```
Comment (daveshanley) [inline cmd/root_command.go:362]: Wait why, did you remove this?

Comment (jacobm-splunk) [reply cmd/root_command.go:362]: It's a duplicated code fragment from line 372. 

Must have been a copy-paste or merge mistake I made.

**Pull request merged**